transformations/syntactically_diverse_paraphrase/transformation.py

```diff
@@ -0,0 +1,55 @@
+import benepar
+import spacy
+import nltk
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+from transformations.syntactically_diverse_paraphrase.sowreap.parse_utils import (
+    Sentence,
+)
+from transformations.syntactically_diverse_paraphrase.sowreap.reap_utils import (
+    reapModel,
+)
+from transformations.syntactically_diverse_paraphrase.sowreap.sow_utils import (
+    sowModel,
+)
+
+
+class ParaphraseSowReap(SentenceOperation):
+    tasks = [TaskType.TEXT_TO_TEXT_GENERATION]
+    languages = ["en"]
+
+    def __init__(self, seed=0, max_outputs=1):
+        super().__init__(seed, max_outputs=max_outputs)
+        self.sow = sowModel("tanyagoyal/paraphrase-sow", max_outputs)
+        self.reap = reapModel("tanyagoyal/paraphrase-reap", max_outputs)
+        self.nlp = spacy.load("en_core_web_sm")
+        try:
+            nltk.data.find(f"models/benepar_en3")
+        except LookupError:
+            benepar.download('benepar_en3')
+
+        if spacy.__version__.startswith("2"):
+            self.nlp.add_pipe(benepar.BeneparComponent("benepar_en3"))
+        else:
+            self.nlp.add_pipe("benepar", config={"model": "benepar_en3"})
+        self.max_outputs = max_outputs
+
+    def generate(self, sentence: str):
+        parse = list(self.nlp(sentence).sents)[0]._.parse_string
```

Comment:
Could you add a comment about what this does?
It seems that you use spacy to obtain the first sentence of a text, however it is not what is done at the end `._.parse_string`.

Reply:
Done